We don't want to remove $PYTHONPATH entries set by modules.
Hence remove the code that seemingly was added to workaround user mistakes.

Author: Alexander Grund (TU Dresden)

diff --git a/spyder/app/start.py b/spyder/app/start.py
index 97b08a600..47873cab2 100644
--- a/spyder/app/start.py
+++ b/spyder/app/start.py
@@ -6,21 +6,8 @@
 # (see spyder/__init__.py for details)
 # -----------------------------------------------------------------------------
 
-# Remove PYTHONPATH paths from sys.path before other imports to protect against
-# shadowed standard libraries.
 import os
 import sys
-if os.environ.get('PYTHONPATH'):
-    for path in os.environ['PYTHONPATH'].split(os.pathsep):
-        if os.name == 'nt' and 'pkgs' in path:
-            # Don't remove pynsist installer entry for 'pkgs' directory
-            continue
-        else:
-            try:
-                sys.path.remove(path.rstrip(os.sep))
-            except ValueError:
-                pass
-
 # Standard library imports
 import ctypes
 import logging
